experiments/experiments.go

```diff
@@ -121,11 +122,11 @@ type ExperimentConfig struct {
 	// StartTimestamp is a float of seconds since the epoch of date and time
```

Comment:
Note that the comment here mentioned that the number in json should actually be float instead of int. That's why I defined the json type as float type instead of int type.

Reply:
Oh good catch :)